sensu-plugins-azurerm.gemspec

```diff
@@ -36,10 +36,16 @@ Gem::Specification.new do |s|
   s.add_development_dependency 'bundler',                   '~> 1.7'
   s.add_development_dependency 'codeclimate-test-reporter', '~> 0.4'
   s.add_development_dependency 'github-markup',             '~> 1.3'
+  s.add_development_dependency 'kitchen-docker',            '~> 2.6'
+  s.add_development_dependency 'kitchen-localhost',         '~> 0.3'
+  # locked to keep ruby 2.1 support, this is pulled in by test-kitchen
+  s.add_development_dependency 'mixlib-shellout',           ['< 2.3.0', '~> 2.2']
   s.add_development_dependency 'pry',                       '~> 0.10'
   s.add_development_dependency 'rake',                      '~> 10.5'
   s.add_development_dependency 'redcarpet',                 '~> 3.2'
-  s.add_development_dependency 'rubocop',                   '~> 0.40.0'
   s.add_development_dependency 'rspec',                     '~> 3.4'
+  s.add_development_dependency 'rubocop',                   '~> 0.40.0'
```

Comment:
we should use the latest rubocop to get the security fix.

Reply:
ya I thought about it, I'd prefer to address them separately if you are OK with that. Those types of updates can be rather noisy and prone to accidental breakage and wanted to minimally get a community supported version out before potentially breaking it.

Reply:
yup, that's fine.

Reply:
I have created a PR and linked it here to address this as well as bumping `sensu-plugin` 2.x, in order to avoid breaking current people I released a 1.0 version prior to implementing these changes.